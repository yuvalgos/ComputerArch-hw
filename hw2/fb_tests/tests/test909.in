r 0x7f59c
r 0x7f580
w 0x7f520
r 0x7f588
w 0x7f53c
w 0x7f578
w 0x7f540
w 0x7f54c
r 0x7f5a0
w 0x7f570
w 0x7f574
w 0x7f538
r 0x7f568
r 0x7f550
r 0x7f540
w 0x7f53c
w 0x7f500
r 0x7f554
w 0x7f534
w 0x7f524
w 0x7f50c
w 0x7f59c
w 0x7f540
r 0x7f5a0
w 0x7f574
w 0x7f548
r 0x7f524
r 0x7f594
r 0x7f504
r 0x7f58c
w 0x7f514
w 0x7f598
w 0x7f574
r 0x7f514
w 0x7f550
w 0x7f51c
r 0x7f59c
w 0x7f508
w 0x7f528
r 0x7f52c
r 0x7f508
