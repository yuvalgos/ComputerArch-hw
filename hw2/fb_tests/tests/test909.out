L1miss=0.854 L2miss=0.571 AccTimeAvg=136.805
